pkg/discord/main.go

```diff
@@ -535,6 +544,122 @@ func sendGithubUsername(data CommandData) {
 	}
 }
 
+type FileRecord struct {
+	airtable.Record
+	Fields struct {
+		Name string
+		Link string
+	}
+}
+
+// Add a file to Airtable tracking
+func trackFile(data CommandData) {
+	fileName := data.Args[0]
+	link := data.Args[1]
+	client := airtable.Client{
+		APIKey: global.AirtableKey,
+		BaseID: brigades[data.GuildID].AirtableBaseID,
+	}
+	file, err := fetchFile(data)
+	if file != nil {
+		if _, err := data.Session.ChannelMessageSend(data.ChannelID, "A file with the name **"+fileName+"** is already tracked: "+file.Fields.Link); err != nil {
+			fmt.Println("error sending message to channel,", err)
+		}
+		return
+	} else if err != nil {
+		if _, err := data.Session.ChannelMessageSend(data.ChannelID, "Failed to check if a file with the name **"+fileName+"** is already tracked."); err != nil {
+			fmt.Println("error sending message to channel,", err)
+		}
+		return
+	}
+	files := client.Table("Files")
```

Comment:
does this create a table every time ?

Reply:
No, just fetches the table named "files". Would be inline(`client.Table("Files").List(...)`) but the airtable library takes the address of the table, so it has to be set to a variable first.

Reply:
gotcha 👌